provider/oauth2/tests.py

```diff
@@ -64,6 +65,7 @@ def _login_and_authorize(self, url_func=None):
         self.assertTrue(self.redirect_url() in response['Location'])
 
 
+@ddt.ddt
```

Comment:
Please add a test case to verify that a `refresh_token` is *not* returned when the response_type is 'token' (and the implicit flow is used).